### PR TITLE
Encrypt organization audit template password

### PR DESCRIPTION
**Pull request by perryr16**
#### Any background context you want to provide?
Audit Template Password was not encrypted in the database.

#### What's this PR do?
Encrypts org.audit_template_password in db
Decrypts password for frontend

#### How should this be manually tested?
from the data page, import audit template buildings.

#### What are the relevant tickets?
#4585

### CODE REVIEW

File: seed/lib/superperms/orgs/migrations/0031_ecrypt_existing_audit_template_passwords [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# Generated by Django 3.2.19 on 2023-05-11 18:49
+from django.db import migrations, transaction
+from seed.utils.encrypt import decrypt, encrypt
+
+@transaction.atomic
+def encrypt_existing_audit_template_passwords(apps, schema_editor):
+    Organization = apps.get_model('orgs', 'Organization')
+
+    for org in Organization.objects.iterator():
+        _, error = decrypt(org.audit_template_password)
```
Comment (axelstudios) [inline seed/lib/superperms/orgs/migrations/0031_ecrypt_existing_audit_template_passwords:10]: Is there any possibility that this field will already be encrypted before this migration?

File: seed/lib/superperms/orgs/migrations/0031_ecrypt_existing_audit_template_passwords [hunk 2]
```diff
@@ -0,0 +1,23 @@
+# Generated by Django 3.2.19 on 2023-05-11 18:49
```
Comment (axelstudios) [inline seed/lib/superperms/orgs/migrations/0031_ecrypt_existing_audit_template_passwords:1]: Fix `ecrypt` typo in migration filename

**Review by axelstudios — CHANGES_REQUESTED** [review]

**Review by kflemin — APPROVED** [review]
works great!

File: seed/lib/superperms/orgs/migrations/0031_ecrypt_existing_audit_template_passwords (hunk 1 above)
Comment (kflemin) [reply seed/lib/superperms/orgs/migrations/0031_ecrypt_existing_audit_template_passwords]: no I don't think so. there has not been an encryption mechanism prior to this PR.

**Review by axelstudios — APPROVED** [review]

**Pull request merged**